service/geopm-service.spec.in

```diff
@@ -48,15 +48,19 @@ BuildRequires: systemd-devel
 %define python_major_version 3
 BuildRequires: python3
 BuildRequires: python3-devel
+BuildRequires: python3-setuptools
 %define python_bin %{__python3}
 
 %if 0%{?suse_version}
+BuildRequires: libdbus-1-3
```

Comment:
I think I was wrong to recommend this in our earlier discussion. There's an OBS linter warning about this:
> geopm-service.x86_64: W: explicit-lib-dependency libdbus-1-3
You must let rpm find the library dependencies by itself. Do not put unneeded
explicit Requires: tags.

Is our actual dependency from geopm-service on `dbus-1`, maybe?

Reply:
I have removed the Requires: section, which is what the linter is complaining about.  I have left the BuildRequires.